lib/eslint-rails/config.rb

```diff
@@ -10,7 +10,7 @@ def self.read(force_default: false)
     CONFIG_PATH = 'config/eslint.json'
     private_constant :CONFIG_PATH
 
-    def initialize(force_default:)
+    def initialize(force_default: raise(ArgumentError, 'force_default is required'))
```

Comment:
I'd prefer it if instead of doing this inline in the declaration, we make this `force_default: nil` and then raise on the first line if it's nil.